packages/bff/src/routers/selfcareRouter.ts

```diff
@@ -101,7 +105,32 @@ const selfcareRouter = (
           error,
           getSelfcareErrorMapper,
           ctx.logger,
-          "Error retrieving institutions"
+          `Error retrieving institutions`
+        );
+        return res.status(errorRes.status).json(errorRes).end();
+      }
+    })
+
+    .get("/tenants/:tenantId/users", async (req, res) => {
+      const ctx = fromBffAppContext(req.ctx, req.headers);
+
+      try {
+        const results = await selfcareService.getInstitutionUsers(
+          unsafeBrandId<TenantId>(req.params.tenantId),
+          req.query.personId
+            ? unsafeBrandId<UserId>(req.query.personId)
+            : undefined,
+          req.query.query as string,
```

Comment:
Is there a way to avoid this cast?

Reply:
yes is no more usefull, solved in this [ commit](https://github.com/pagopa/interop-be-monorepo/pull/986/commits/bf70a9ffbd23fad276e084bb62f04a094c361940)